google-ads-examples/src/main/java/com/google/ads/googleads/examples/errorhandling/HandlePartialFailure.java

```diff
@@ -95,6 +95,26 @@ public static void main(String[] args) throws InvalidProtocolBufferException {
   /** Runs the example. */
   private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId)
       throws InvalidProtocolBufferException {
+    MutateAdGroupsResponse response = createAdGroups(googleAdsClient, customerId, campaignId);
+
+    // Checks for existence of any partial failures in the response.
+    if (checkIfPartialFailureErrorExists(response)) {
+      System.out.println("Partial failures occurred. Details will be shown below.");
+    } else {
+      System.out.println("All operations completed successfully. No partial failures to show.");
+      return;
+    }
+
+    // Finds the failed operations by looping through the results.
+    printResult(response);
```

Comment:
nit: This method actually goes through the response's _results_ (plural), so I'd name this `printResults`.